noon-progress.py

```diff
@@ -30,4 +31,4 @@
 # wait until it is time to post
 time.sleep(noonposix - nowposix)
 
-r = requests.post(config.webhook_url, data=payload_start)
\ No newline at end of file
+r = requests.post(config.webhook_url, data=payload_start)

```

Comment:
Add newline at end of file
```suggestion
r = requests.post(config.webhook_url, data=payload_start)

```